js/demo_policy/test_scenarios.ts

```diff
@@ -64,6 +66,56 @@ export function handleDnsLookupFailure(): Validation.ValidationResponse {
   );
 }
 
+/**
+ * Handles OCI manifest lookup success scenario
+ */
+export function handleOciManifestSuccess(): Validation.ValidationResponse {
+  const image = 'docker.io/library/busybox:1.36';
+  let manifest: OciImageManifestResponse | null = null;
+  try {
+    manifest = Manifest.getOCIManifest(image);
+  } catch (e) {
+    console.error('OCI manifest lookup failed:', e);
+  }
+  const manifestType = manifest?.image ? 'image' : manifest?.index ? 'index' : '';
+  return new Validation.ValidationResponse(
+    !!manifest && manifestType !== '',
+    manifestType ? undefined : 'Failed to retrieve OCI manifest',
+    undefined,
+    undefined,
+    { manifestType },
+  );
+}
+
+/**
+ * Handles OCI manifest lookup failure scenario
+ */
+export function handleOciManifestFailure(): Validation.ValidationResponse {
+  const image = 'registry.testing.lan/nonexistent-image:1.0.0';
+  let manifest: OciImageManifestResponse | null = null;
+  try {
+    manifest = Manifest.getOCIManifest(image);
+  } catch (e) {
+    console.error('OCI manifest lookup failed:', e);
+    return new Validation.ValidationResponse(
+      false,
+      `OCI manifest lookup failed: ${e}`,
+      undefined,
+      undefined,
+      { manifestType: '' },
+    );
+  }
+  // If manifest is retrieved, treat as failure
```

Comment:
I would drop the try-catch and the happy path, and use an image such as `example.test/nonexistent-image:1.0.0`, which should never resolve (see https://www.rfc-editor.org/rfc/rfc2606.html)